api/config/config_test.go

```diff
@@ -311,3 +316,318 @@ func TestStandardTransformerConfig_ToFeastStorageConfigs(t *testing.T) {
 		})
 	}
 }
+
+func setupNewEnv(envMaps ...map[string]string) error {
+	os.Clearenv()
+
+	var err error
+	for _, envMap := range envMaps {
+		for key, val := range envMap {
+			err = os.Setenv(key, val)
+			if err != nil {
+				return err
+			}
+		}
+	}
+	return nil
+}
+
+func TestLoad(t *testing.T) {
```

Comment:
I added this unit test to test the parsing of configs from .yaml files to the actual Go config struct because I realised I left this out the last time I refactored the Merlin API server to consume configs from .yaml files.